src/api/world.ts

```diff
@@ -234,6 +249,15 @@ export const updateWorldAdvancedSettings = async (
   )(await createFirestoreWorldAdvancedInput(world, user));
 };
 
+export const updateWorldScheduleSettings = async (
+  world: WithId<WorldScheduleSettings>
+) => {
+  return await httpsCallable(
+    FIREBASE.functions,
+    "world-updateWorld"
```

Comment:
[NOTE] I really think there is a way to put all these function constants in one place for better overview, not for this PR though.